.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
Can you also add `deps` and `_build` as well as variations of the built executable as created by the build script?